messaging/src/test/java/org/springframework/security/messaging/context/SecurityContextChannelInterceptorTests.java

```diff
@@ -138,13 +139,6 @@ public void preSendUserNotSet() {
 		assertAnonymous();
 	}
 
-	// SEC-2845
-	@Test
```

Comment:
I don't think we want to remove tests, especially those linked to an issue. Is there a reason for removing this one?

Reply:
Oh! I didn't mention that in the commit comments. 
Sorry.
I removed it deliberately just because this test is really a plain copy of `preSendUserNotSet()`.
Perhaps some logic is missed because their names are different, but that already not relevant to this PR.

Well, I can restore it if you insist, but let's see if this test can be improved somehow to reflect its purpose or so!

Reply:
According the name of this test, if we do `setAnonymousAuthentication()`, this test becomes identical to the `preSendUsesCustomAnonymous()`. Just because that one also does set a user into a message to intercept.
So, one of this tests is redundant this or other way.